website/docs/docs/dbt-versions/core-upgrade/01-upgrading-to-v1.8.md

```diff
@@ -21,6 +21,8 @@ dbt Cloud is going "versionless." This means you'll automatically get early acce
 
 Select ["Keep on latest version"](/docs/dbt-versions/upgrade-dbt-version-in-cloud#keep-on-latest-version) in your development, staging, and production [environments](/docs/deploy/deploy-environments) to access to everything in dbt Core v1.8 and more.
 
+For the dbt Cloud Admin API and Terraform, use the value `versionless` to configure your environment. 
```

Comment:
not sure if I should specify that users can input the value in the dbt_version key ?

Reply:
Yes, I think that's a good clarification - "To upgrade an environment via Admin API or Terraform, set `dbt_version` to the string `"versionless"`